tests/useInfiniteQuery.test.ts

```diff
@@ -3,7 +3,7 @@ import { useInfiniteQuery } from "../src/useInfiniteQuery";
 import { noop, infiniteFetcher, flushPromises } from "./utils";
 
 jest.mock("vue", () => {
-  const vue = jest.requireActual("vue");
+  const vue = jest.requireActual("vue-demi");
```

Comment:
I believe that you can leave imports in tests as-is, this should solve your issue with failing tests.